pubtools/pulplib/_impl/client/client.py

```diff
@@ -340,6 +340,25 @@ def search_distributor(self, criteria=None):
         """
         return self._search(Distributor, "distributors", criteria=criteria)
 
+    def search_task(self, criteria=None):
+        """Search the tasks matching the given criteria.
+
+        Args:
+            criteria (:class:`~pubtools.pulplib.Criteria`)
+                A criteria object used for this search.
+                If None, search for all tasks.
+
+        Returns:
+            Future[:class:`~pubtools.pulplib.Page`]
+                A future representing the first page of results.
+
+                Each page will contain a collection of
+                :class:`~pubtools.pulplib.Task` objects.
+
+        .. versionadded:: 2.18.0
```

Comment:
I think this versionadded value is obsolete already, there were probably some releases after you started working on it.

```suggestion
        .. versionadded:: 2.20.0
```

Reply:
Thanks. Should the version here be 2.19.0 instead of 2.20.0? I see the latest version as of now is 2.18.0, unless I'm missing anything.

Reply:
Yeah, my mistake.  I was thinking the "Added `requires` and `provides`" change had already been released.